Makefile

```diff
@@ -50,50 +40,43 @@ clean: ## Make a clean source tree
 	-find website -name '.sass-cache' -type d -exec rm -fr {} +
 	-find website -name '_site' -type d -exec rm -fr {} +
 	-find website -name 'build' -type d -exec rm -fr {} +
-	-$(SA) $(ENV) && make -C docs clean
-	-$(SA) $(ENV) && make -C etc clean
+	-make -C docs clean
+	-make -C etc clean
 
 lint: ## Check code style
-	$(SA) $(ENV) && pre-commit run --all-files
-
-nuke: ## Make clean + remove conda env
-	-conda env remove -n $(ENV) -y
+	pre-commit run --all-files
 
-dev: ## Make a server in jupyter_websocket mode
-	$(SA) $(ENV) && python enterprise_gateway
+run-dev: test-install-wheel ## Make a server in jupyter_websocket mode
+	python enterprise_gateway
 
 docs: ## Make HTML documentation
-	$(SA) $(ENV) && make -C docs requirements html
+	make -C docs requirements html
 
 kernelspecs:  kernelspecs_all kernelspecs_yarn kernelspecs_conductor kernelspecs_kubernetes kernelspecs_docker kernel_image_files ## Create archives with sample kernelspecs
 kernelspecs_all kernelspecs_yarn kernelspecs_conductor kernelspecs_kubernetes kernelspecs_docker kernel_image_files:
 	make VERSION=$(VERSION) TAG=$(TAG) SPARK_VERSION=$(SPARK_VERSION) -C  etc $@
 
-install: ## Make a conda env with dist/jupyter_enterprise_gateway-*.whl and dist/jupyter_enterprise_gateway-*.tar.gz installed
-	-conda env remove -y -n $(ENV)-install
-	conda create -y -n $(ENV)-install python=3 pip
-	$(SA) $(ENV)-install && \
-		pip install dist/jupyter_enterprise_gateway-*.whl && \
-			jupyter enterprisegateway --help && \
-			pip uninstall -y jupyter_enterprise_gateway
-	conda env remove -y -n $(ENV)-install
-
-	conda create -y -n $(ENV)-install python=3 pip
-	$(SA) $(ENV)-install && \
-		pip install dist/jupyter_enterprise_gateway-*.tar.gz && \
-			jupyter enterprisegateway --help && \
-			pip uninstall -y jupyter_enterprise_gateway
-	conda env remove -y -n $(ENV)-install
+test-install: test-install-wheel test-install-tar ## Install and minimally run EG with the wheel and tar distributions
+
+test-install-wheel:
+	pip uninstall -y jupyter_enterprise_gateway
+	pip install dist/jupyter_enterprise_gateway-*.whl && \
+		jupyter enterprisegateway --help
+
+test-install-tar:
+	pip uninstall -y jupyter_enterprise_gateway
+	pip install dist/jupyter_enterprise_gateway-*.tar.gz && \
+		jupyter enterprisegateway --help
 
 bdist: lint
 	make $(WHEEL_FILE)
 
 $(WHEEL_FILE): $(WHEEL_FILES)
-	$(SA) $(ENV) && python setup.py bdist_wheel $(POST_SDIST) \
+	python setup.py bdist_wheel $(POST_SDIST) \
```

Comment:
Invoking `setup.py` directly is deprecated.  This would be `pip install build && python -m build --wheel .` (use `--sdist` below)

Reply:
Thanks @blink1073!  My latest commit updates these to use `build` and `twine`.  I also fixed up the targets to utilize target dependencies correctly.

Reply:
Thanks!